testsupport/appstudio/api/v1alpha1/README.adoc

```diff
@@ -0,0 +1,8 @@
+= HOW-TO
+
+Copy content from:
+- https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/environment_types.go
+- https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/snapshotenvironmentbinding_types.go
```

Comment:
we don't use `snapshotenvironmentbinding_types.go`, right?
```suggestion
- https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/application_types.go
```

Reply:
https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/snapshotenvironmentbinding_types.go contains the definition for [EnvVarPair](https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/snapshotenvironmentbinding_types.go#L96-L103) which is used in [EnvironmentConfiguration](https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/environment_types.go#L112-L115)

Reply:
ah ok, so you put everything into one file? I missed that :man_facepalming: